packages/styles/src/css.js

```diff
@@ -123,25 +125,25 @@ export const responsive = (styles = {}) => {
  * Enhances the (create-system enhanced) CSS function to account for
  * scale functions within the Style system.
  *
- * @param {any} args The styles to compile.
- * @returns {string} The compiled styles.
+ * @param {TemplateStringsArray | import('create-emotion').Interpolation<undefined>} template
+ * @param {import('create-emotion').Interpolation<undefined>[]} args The styles to compile.
+ * @returns {ReturnType<compile>} The compiled styles.
  */
-export function css(...args) {
-	const [arg, ...rest] = args;
-
-	if (is.plainObject(arg)) {
-		return compile(getScaleStyles(responsive(arg)));
+export function css(template, ...args) {
+	if (is.objectInterpolation(template)) {
+		return compile(getScaleStyles(responsive(template)));
 	}
 
-	if (is.array(arg)) {
-		for (let i = 0, len = arg.length; i < len; i++) {
-			const n = arg[i];
-			if (is.plainObject(n)) {
-				arg[i] = getScaleStyles(responsive(n));
+	if (is.array(template)) {
+		for (let i = 0, len = template.length; i < len; i++) {
+			const n = template[i];
+			if (is.objectInterpolation(n)) {
+				template[i] = getScaleStyles(responsive(n));
 			}
 		}
-		return compile(...[arg, ...rest]);
+		return compile(template, ...args);
 	}
 
-	return compile(...args);
+	// @ts-ignore
+	return compile(template, ...args);
 }
```

Comment:
@ItsJonQ Can you confirm this still matches your intention or am I missing something? In particular I'm concerned about `is.objectInterpolation` whether that was exactly your intention here or if there was something else that might get passed in your were looking for.

Reply:
This looks correct! Thank you @saramarcondes ❤️